xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Game.h

```diff
@@ -29,18 +29,24 @@ namespace addon
 ///
 
 //==============================================================================
-/// \defgroup cpp_kodi_addon_game_Defs Definitions, structures and enumerators
-/// \ingroup cpp_kodi_addon_game
+/// @defgroup cpp_kodi_addon_game_Defs Definitions, structures and enumerators
```

Comment:
Is `@` preferred over `\`?

Reply:
For me yes :grinning:, I look to use with addon header the `@`.

Not sure why, the `\` makes me aggressive :smirk:, but mainly about text search. Normally a `\` says for me to prevent a following character e.g. `"some C code text with \"value\""` or to set a text special value e.g. `\n`